app/components/Inputs/AssetInput/AssetInput.jsx

```diff
@@ -1,79 +1,44 @@
 // @flow
 import React from 'react'
-import classNames from 'classnames'
-import { omit, noop } from 'lodash-es'
 
-import SelectInput from '../SelectInput'
+import StyledReactSelect from '../StyledReactSelect/StyledReactSelect'
 import styles from './AssetInput.scss'
 
 type Props = {
-  className?: string,
-  value?: string,
   symbols: Array<SymbolType>,
-  onChange: Function,
-  onFocus: Function,
-  onBlur: Function
+  onChange: Function
 }
 
 type State = {
-  typingValue: string
+  selectedAsset: Object
 }
 
 export default class AssetInput extends React.Component<Props, State> {
-  static defaultProps = {
-    onChange: noop,
-    onFocus: noop,
-    onBlur: noop
-  }
+  parsedSymbols = this.props.symbols.map(symbol => ({
```

Comment:
I think its better if this was a function that would get symbols and check if its an array and then return a mapped result (outside of the class)

Reply:
👍